nava/params.py

```diff
@@ -12,6 +12,7 @@
 SOUND_FILE_EXIST_ERROR = "Given sound file doesn't exist."
 SOUND_FILE_PATH_TYPE_ERROR = "Sound file's path should be a string."
 SOUND_ID_EXIST_ERROR = "Given sound id doesn't exist."
+LOOP_ASYNC_ERROR = "The loop option can only be used in async mode."
```

Comment:
@sepandhaghighi 
I think we should use a more direct error message, pointing out to the exact variable names. Something like below:
```python
LOOP_ASYNC_ERROR = "`loop` can not be set true when `async_mode` is False."
```

Reply:
Good point, I will hande it

Reply:
fix in https://github.com/openscilab/nava/pull/31/commits/33d1f5c7d5e6d3f739000af85cc75a29c9966e0c